addon/index.js

```diff
@@ -10,8 +10,8 @@ import { fillable }    from './-private/properties/fillable';      export { fill
 import { hasClass }    from './-private/properties/has-class';     export { hasClass };
 import { is }          from './-private/properties/is';            export { is };
 import { isHidden }    from './-private/properties/is-hidden';     export { isHidden };
+import { isPresent }      from './-private/properties/is-present';        export { isPresent };
```

Comment:
Small thing but could you fix the spacing to make it consistent with the other lines?

Reply:
Oh yeah, absolutely...thx.